opm/simulators/wells/GasLiftSingleWellGeneric.cpp

```diff
@@ -817,6 +819,7 @@ getLimitedRatesFromRates_(const BasicRates& rates) const
                          rates.gas,
                          water_rate,
                          rates.water,
+                         rates.bhp,
```

Comment:
The name `LimitedRates` seems to indicate to the reader that it contains limited rates. But now it also contains `bhp` which is a pressure.